src/shared/types/domain-types.ts

```diff
@@ -340,6 +340,7 @@ export type Song = {
     itemType: LibraryItem.SONG;
     lastPlayedAt: null | string;
     lyrics: null | string;
+    mbzId: null | string;
```

Comment:
Given that this may be a recording id _or_ track id, it would probably be better to just have both? That way you don't have to disambiguate later.